features/steps/step_selenium_api_token.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: UTF-8 -*-
+
+# @mark.steps
+# ----------------------------------------------------------------------------
+# STEPS:
+# ----------------------------------------------------------------------------
+from behave import given, when, then
+import time
+
+@given('we are on Hipchat Home Page')
+def step_impl(context):
```

Comment:
Please, do not copy-paste steps - reuse them https://en.wikipedia.org/wiki/Don%27t_repeat_yourself